docker/docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: "3.9"  # optional since v1.27.0
+services:
+  tracking:
+    build: .
+    args:
+        VERSION: ${GITHUB_REF_NAME}
+    entrypoint: ["mlflow", "server"]
```

Comment:
Is this file used in the workflow added in https://github.com/mlflow/mlflow/pull/6735?

Reply:
you mean #6732? No, it isn't and has been deleted.

Reply:
yes, #6732